src/hooks/useConfig.tsx

```diff
@@ -1,41 +0,0 @@
-import { ComponentChildren, createContext, h, JSX } from 'preact';
-import { StateUpdater, useContext, useState } from 'preact/hooks';
-import type { ConsentManagerConfig } from '@transcend-io/airgap.js-types';
-
-/**
- * Config context
- */
-interface TConfigContext {
-  /** The config */
-  config: Required<ConsentManagerConfig>;
-  /** Set new config */
-  setConfig: StateUpdater<Required<ConsentManagerConfig>>;
-}
-
-/**
- * Context to pass Emotion through the app
- */
-export const ConfigContext = createContext<TConfigContext>(
-  {} as TConfigContext,
-);
-
-export const ConfigProvider = ({
```

Comment:
i removed this due to reducing complexity - there are so few layers to this project that adding a global state variable is not really necessary. it made this code more complex and makes the other react components slightly harder to unit test (once we add unit tests)